CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ if (${CMAKE_CXX_COMPILER_ID} STREQUAL "MSVC")
   set(RELEASE_FLAGS, "/GS /GL /W3 /Gy /Zi /O2")
   set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 else()
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
```

Comment:
Well, it doesn't work. If I add it to those variables, it doesn't seem to pick up the flag, cue a vomit of warnings about c++11 features.